pkg/helpers/sa_syncer.go

```diff
@@ -155,7 +155,7 @@ func applyKubeconfigSecret(ctx context.Context, templateKubeconfig *rest.Config,
 		},
 		AuthInfos: map[string]*clientcmdapi.AuthInfo{
 			"user": {
-				TokenFile: "token",
+				Token: string(token),
```

Comment:
why make this change?

Reply:
Because the path is not right.

It could be fixed in this way:
```
TokenFile: "/var/run/secrets/hub/token"
```